node/services/config.go

```diff
@@ -40,5 +42,8 @@ func (cfg *Config) trustedPeer() (*peer.AddrInfo, error) {
 }
 
 func (cfg *Config) trustedHash() (tmbytes.HexBytes, error) {
+	if cfg.TrustedHash == "" {
```

Comment:
Seems generally leaky to compare against `""` (which will be a run-time failure if the implementation of `TrustedHash` changes) instead of using a method on `cfg` or `TrustedHash` (which would be a compile-time failure). But whatever.

Reply:
Wait? It is not leaky to compare string against empty string? If the implementation of `TrustedHash` changes then there will be a compile-time error that you cannot compare against an empty string.